src/pages/BorrowerProfile/BorrowerProfile.vue

```diff
@@ -275,7 +294,31 @@ export default {
 				return `${loanUse}\n\n${this.description.substring(0, 120)}...`;
 			}
 			return 'For the borrower\'s privacy, this loan has been made anonymous.';
+		},
+		showUrgencyExp() {
+			return this.hasThreeDaysOrLessLeft && this.isUrgencyExpVersionShown;
 		}
 	},
+	created() {
+		/*
+		 * Experiment Initializations
+		*/
+		this.apollo.query({ query: experimentAssignmentQuery, variables: { id: 'lend_urgency' } })
+			.then(result => {
+				const urgencyEXP = result?.data?.experiment;
+				// Only track event if loan is expiring soon
+				this.isUrgencyExpVersionShown = urgencyEXP.version === 'shown';
```

Comment:
Couldn't this go within the 3 days condition? Why change the experience if you're not tracking that session?

Reply:
not a deal breaker, just kinda odd that some folks would see this without reason.

Reply:
I dont think it makes a difference. The experience won't be changed unless has 3 days or less (based on the computed prop above)

Reply:
Ok, I see now you have the `showUrgencyExp` prop with that additional check so all clear. Thanks!

Reply:
👍  I'm gonna have to refactor this anyway, since this same experiment key will also drive the homepage part of this.